checker/validate.mli

```diff
@@ -0,0 +1,9 @@
+(************************************************************************)
+(*  v      *   The Coq Proof Assistant  /  The Coq Development Team     *)
```

Comment:
What's the point of all these copyright headers. You already get copyright on everything you produce in Europe anyway. You can add such headers to generated source code in a release, if you want, but adding it yourself manually just adds noise and wastes time.

Reply:
> What's the point of all these copyright headers.

That's the recommended practice by the FSF and it may sound a bit outdated but it is still useful to trace the origin of a single file (even if people with malign intent can very well strip this part).

> You already get copyright on everything you produce in Europe anyway.

That doesn't apply just to European countries but to all countries that signed the Berne convention.

> You can add such headers to generated source code in a release, if you want, but adding it yourself manually just adds noise and wastes time.

Right now we don't generate any source code for releases. Moreover people get the source code directly from GitHub more and more and from released tarballs less and less.